server/server.go

```diff
@@ -606,9 +608,14 @@ func New(config *Config) (*Server, error) {
 	s.restore()
 	s.cleanupSandboxesOnShutdown()
 
+	bindAddress, err := knet.ChooseBindAddress(net.IP{0, 0, 0, 0})
+	if err != nil {
+		return nil, err
+	}
+
 	// Prepare streaming server
 	streamServerConfig := streaming.DefaultConfig
-	streamServerConfig.Addr = "0.0.0.0:10101"
+	streamServerConfig.Addr = bindAddress.String() + ":10101"
```

Comment:
We'll likely provide a way for people to configure the port (and bind address) but since nobody can actually configure it we're leaving it hard-coded for now.